### PR TITLE
update the timeout error

### PR DESCRIPTION
**Pull request by HugoPct**
### What does this PR do?
update the timeout error section for synthetic tests

### Preview

https://docs-staging.datadoghq.com/<BRANCH_NAME>/<PATH>

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/api_tests/http_tests.md [hunk 1]
```diff
@@ -213,8 +213,9 @@ These reasons include the following:
 
 `TIMEOUT`
 : The request couldn't be completed in a reasonable time. Two types of `TIMEOUT` can happen:
-  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the timeout happened at the TCP socket connection level. 
-  - `TIMEOUT: Retrieving the response couldn’t be completed in a reasonable time.` indicates that the timeout happened on the overall run (which includes TCP socket connection, data transfer, and assertions).
+  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the request duration hit the timeout defined in the test (default is set to 60s).
```
Comment (peculiarism) [inline content/en/synthetics/api_tests/http_tests.md:216]:
```suggestion
  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the request duration hit the test defined timeout (default is set to 60s). 
```
Should we link to the API docs for increasing the timeout value?

File: content/en/synthetics/api_tests/http_tests.md [hunk 2]
```diff
@@ -213,8 +213,9 @@ These reasons include the following:
...
+  Only completed subparts are displayed in the network waterfall. Meaning that in the case there is only the `Total response time` displayed, the timeout occurred during the DNS resolution.
```
Comment (peculiarism) [inline content/en/synthetics/api_tests/http_tests.md:217]:
```suggestion
  For each request only the completed stages for the request are displayed in the network waterfall. For example in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```

File: content/en/synthetics/api_tests/http_tests.md [hunk 3]
```diff
@@ -213,8 +213,9 @@ These reasons include the following:
...
+  - `TIMEOUT: Overall test execution couldn't be completed in a reasonable time.` indicates that the request and assertions duration hit the maximum duration that is set to 60.5s.
```
Comment (peculiarism) [inline content/en/synthetics/api_tests/http_tests.md:218]:
```suggestion
  - `TIMEOUT: Overall test execution couldn't be completed in a reasonable time.` indicates that the request and assertions duration hit the maximum duration (60.5s).
```

**Review by peculiarism — COMMENTED** [review]
Couple of comments, might be worth getting some docs review though for grammar as mine is definitely not great.

File: content/en/synthetics/api_tests/http_tests.md (hunk 2 above)
Comment (HugoPct) [reply content/en/synthetics/api_tests/http_tests.md:217]: stages - that's the word
thank you!

File: content/en/synthetics/api_tests/http_tests.md (hunk 1 above)
Comment (HugoPct) [reply content/en/synthetics/api_tests/http_tests.md]: timeout for API tests is by default set to 60s and it is also the max value. Users can only reduce it.
looks like timeout are not in the docs. I'll add it too

File: content/en/synthetics/api_tests/dns_tests.md [hunk 4]
```diff
@@ -149,9 +150,10 @@ These reasons include the following:
 : The configuration of the test is invalid (for example, a typo in the URL).
 
 `TIMEOUT`
-: The request couldn't be completed in a reasonable time. Two types of `TIMEOUT` can happen.
-  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the timeout happened at the TCP socket connection level.
-  - `TIMEOUT: Retrieving the response couldn’t be completed in a reasonable time.` indicates that the timeout happened on the overall run (which includes TCP socket connection, data transfer, and assertions).
+: The request couldn't be completed in a reasonable time. Two types of `TIMEOUT` can happen:
+  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the request duration hit the test defined timeout (default is set to 60s). 
+  For each request only the completed stages for the request are displayed in the network waterfall. For example in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```
Comment (sarina-dd) [inline content/en/synthetics/api_tests/dns_tests.md:155]:
```suggestion
  For each request only the completed stages for the request are displayed in the network waterfall. For example, in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```

File: content/en/synthetics/api_tests/http_tests.md [hunk 5]
```diff
@@ -213,8 +214,9 @@ These reasons include the following:
 
 `TIMEOUT`
 : The request couldn't be completed in a reasonable time. Two types of `TIMEOUT` can happen:
-  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the timeout happened at the TCP socket connection level. 
-  - `TIMEOUT: Retrieving the response couldn’t be completed in a reasonable time.` indicates that the timeout happened on the overall run (which includes TCP socket connection, data transfer, and assertions).
+  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the request duration hit the test defined timeout (default is set to 60s). 
+  For each request only the completed stages for the request are displayed in the network waterfall. For example in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```
Comment (sarina-dd) [inline content/en/synthetics/api_tests/http_tests.md:218]:
```suggestion
  For each request only the completed stages for the request are displayed in the network waterfall. For example, in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```

File: content/en/synthetics/api_tests/ssl_tests.md [hunk 6]
```diff
@@ -161,8 +163,9 @@ These reasons include the following:
 
 `TIMEOUT`
 : The request couldn't be completed in a reasonable time. Two types of `TIMEOUT` can happen:
-  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the timeout happened at the TCP socket connection level.
-  - `TIMEOUT: Retrieving the response couldn’t be completed in a reasonable time.` indicates that the timeout happened on the overall run (which includes TCP socket connection, data transfer, and assertions).
+  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the request duration hit the test defined timeout (default is set to 60s). 
+  For each request only the completed stages for the request are displayed in the network waterfall. For example in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```
Comment (sarina-dd) [inline content/en/synthetics/api_tests/ssl_tests.md:167]:
```suggestion
  For each request only the completed stages for the request are displayed in the network waterfall. For example, in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```

File: content/en/synthetics/api_tests/tcp_tests.md [hunk 7]
```diff
@@ -144,8 +145,9 @@ These reasons include the following:
 
 `TIMEOUT`
 : The request couldn't be completed in a reasonable time. Two types of `TIMEOUT` can happen:
-  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the timeout happened at the TCP socket connection level.
-  - `TIMEOUT: Retrieving the response couldn’t be completed in a reasonable time.` indicates that the timeout happened on the overall run (which includes TCP socket connection, data transfer, and assertions).
+  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the request duration hit the test defined timeout (default is set to 60s). 
+  For each request only the completed stages for the request are displayed in the network waterfall. For example in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```
Comment (sarina-dd) [inline content/en/synthetics/api_tests/tcp_tests.md:149]:
```suggestion
  For each request only the completed stages for the request are displayed in the network waterfall. For example, in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```

File: content/en/synthetics/api_tests/udp_tests.md [hunk 8]
```diff
@@ -145,8 +145,9 @@ These reasons include the following:
 
 `TIMEOUT`
 : The request couldn't be completed in a reasonable time. Two types of `TIMEOUT` can happen:
-  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the timeout happened at the TCP socket connection level.
-  - `TIMEOUT: Retrieving the response couldn’t be completed in a reasonable time.` indicates that the timeout happened on the overall run (which includes TCP socket connection, data transfer, and assertions).
+  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the request duration hit the test defined timeout (default is set to 60s). 
+  For each request only the completed stages for the request are displayed in the network waterfall. For example in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```
Comment (sarina-dd) [inline content/en/synthetics/api_tests/udp_tests.md:149]:
```suggestion
  For each request only the completed stages for the request are displayed in the network waterfall. For example, in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```

File: content/en/synthetics/api_tests/websocket_tests.md [hunk 9]
```diff
@@ -167,8 +167,9 @@ These reasons include the following:
 
 `TIMEOUT`
 : The request couldn't be completed in a reasonable time. Two types of `TIMEOUT` can happen:
-  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the timeout happened at the TCP socket connection level. 
-  - `TIMEOUT: Retrieving the response couldn’t be completed in a reasonable time.` indicates that the timeout happened on the overall run (which includes TCP socket connection, data transfer, and assertions).
+  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the request duration hit the test defined timeout (default is set to 60s). 
+  For each request only the completed stages for the request are displayed in the network waterfall. For example in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```
Comment (sarina-dd) [inline content/en/synthetics/api_tests/websocket_tests.md:171]:
```suggestion
  For each request only the completed stages for the request are displayed in the network waterfall. For example, in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```

File: content/en/synthetics/multistep.md [hunk 10]
```diff
@@ -254,8 +254,9 @@ A test is considered `FAILED` if a step does not satisfy one or several assertio
 
 `TIMEOUT`
 : The request couldn't be completed in a reasonable time. Two types of `TIMEOUT` can happen:
-  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the timeout happened at the TCP socket connection level.
-  - `TIMEOUT: Retrieving the response couldn’t be completed in a reasonable time.` indicates that the timeout happened on the overall run (which includes TCP socket connection, data transfer, and assertions).
+  - `TIMEOUT: The request couldn’t be completed in a reasonable time.` indicates that the request duration hit the test defined timeout (default is set to 60s). 
+  For each request only the completed stages for the request are displayed in the network waterfall. For example in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```
Comment (sarina-dd) [inline content/en/synthetics/multistep.md:258]:
```suggestion
  For each request only the completed stages for the request are displayed in the network waterfall. For example, in the case of `Total response time` only being displayed, the timeout occurred during the DNS resolution.
```

**Review by sarina-dd — COMMENTED** [review]
Small comma suggestions

**Review by sarina-dd — APPROVED** [review]

**Comment by HugoPct** [comment]
can be merged

**Review by cswatt — APPROVED** [review]

**Pull request merged**